crates/proto/src/rustls/tls_client_stream.rs

```diff
@@ -31,17 +31,19 @@ pub type TlsClientStream<S> =
 /// # Arguments
 ///
 /// * `name_server` - IP and Port for the remote DNS resolver
+/// * `bind_addr` - IP and port to connect from
 /// * `dns_name` - The DNS name, Subject Public Key Info (SPKI) name, as associated to a certificate
 #[allow(clippy::type_complexity)]
 pub fn tls_client_connect<S: Connect>(
     name_server: SocketAddr,
+    bind_addr: Option<SocketAddr>,
```

Comment:
I'm trying to decide if we need a new method here to avoid a breaking change.

Reply:
I've added tls_client_connect_with_bind_addr.